### ISSUE TITLE
Can't invoke click() handler on button within suggestion

### ISSUE THREAD

**Issue by belov91** [twitter/typeahead.js#1043]
```javascript 
$(document).ready(function(){
        $('#search2').typeahead({
                hint: true,
                highlight: true
            },
            {
                name: 'products',
                displayKey: 'name',
                source: function (query, process) {
                    return $.post("php/Products.php", { action:'sell-search',search_value: query}, function (data) {
                        process(JSON.parse(data));
                    });
                },
                templates: {
                    empty: [
                        '<div class="empty-message text-danger">',
                        '<h3>Няма намерена стока</h3>',
                        '</div>'
                    ].join('\n'),
                    suggestion: Handlebars.compile(
                        '<div class="row">' +
                            '<div class="col-xs-8"><div>{{name}}</div><div class="small text-success">{{category}}</div></div>' +
                            '<div class="col-xs-3">{{price_retail}} <span class="x-small">лв/{{measure}}<span></div>'+
                            '<div class="col-xs-1"><button class="btn btn-sm btn-warning float-right products-info "><span class="glyphicon glyphicon-info-sign"></span></button></div>' +
                        '</div>')
                }
            });
});

    $(document).on('click','.products-info', function(e){
        e.stopPropagation();
        console.log("Cliicked");
        console.log('Inside link');
        alert('test');
        return false;
    });
```

This is my code, I am trying to implement clickable suggestion ( by clickable I mean, when the user clicks the '.products-info' button of the suggestion an alert will be launched up, AND the drop down menu with the suggestions will NOT hide). 

What it heppans now is: 
1. hide drop down menu 
2. console.log("Cliicked");
3. console.log('Inside link');
4. alert('test');

P.S. I am using  typeahead.js 0.10.5.

**Comment by jharding** [comment]
I believe this is fixed in v0.11. Let me know if it isn't.

**Comment by slmdev23** [comment]
It is still not working in v0.11.
